### PR TITLE
Hardmode added

### PR DESCRIPTION
**Pull request by vaibhavviking**
Now there is a hardmode where the controls for the game are inverted. Hardmode can be toggled on/off anytime.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,13 +6,15 @@ const startBtn = document.querySelector(".btn-start");
...
+const inv = document.querySelector(".btn-inv");
+const desc=document.querySelector(".desc");
 
 const HIGHSCORESTRING = "highscore";
 const STARTSPEED = 200;
 const SPEEDINCREMENT = 5;
 const MINSPEED = 25;
 const MAX_DIRECTION_BUFFER_LENGTH = 3;
-
+var hardmode = 0;
```
Comment (indremak) [inline index.js]: Please change `var` to `let`, use camelCase for variable names.

File: index.js [hunk 2]
```diff
@@ -115,17 +117,36 @@ function showScore() {
 //keycode: up 38, down 40, right 39, left 37
 // w: 87, a: 65, s: 83, d: 68
 function handleInput(e) {
-  if (e.keyCode === 39 || e.keyCode === 68) {
-    addNewDirection("right");
+  if (e.keyCode === 82) {
+    init();
+  }else if (e.keyCode === 39 || e.keyCode === 68) {
+    if(hardmode==0){
```
Comment (indremak) [inline index.js]: It would be better if  you replace all double equals `==`  with triple equals `===`.

File: index.js [hunk 3]
```diff
@@ -115,17 +117,36 @@ function showScore() {
...
-    addNewDirection("down");
-  } else if (e.keyCode === 82) {
-    init();
+    if(hardmode==0){
+      addNewDirection("down");
+    }else{
+      addNewDirection("up");
+    }
   }
+  //  else if (e.keyCode === 82) {
```
Comment (indremak) [inline index.js]: Please don't leave commented out code.

**Review by indremak — CHANGES_REQUESTED** [review]

**Comment by vaibhavviking** [comment]
I have made the required changes. Please review them.

**Pull request merged**